src/app/ticker/ticker.component.html

```diff
@@ -8,17 +8,13 @@
     <span *ngIf="!fund && campaign.amountRaised > 0 && !campaignInFuture" ><em class="c-ticker__value c-ticker__value--emphasised">{{ campaign.amountRaised | currency : '£' : 'symbol' : '1.0-0' }}</em> raised inc. Gift Aid</span>
     <span *ngIf="!fund && (campaign.amountRaised == 0 || campaignInFuture)">Opens in <em  class="c-ticker__value c-ticker__value--emphasised">{{ campaign.startDate | timeLeft }}</em></span>
   </div>
-  <div class="c-ticker__lists">
+  <div *ngIf="!fund" class="c-ticker__lists">
     <ul class="c-ticker__list" *ngFor="let index of [0,1,2,3]">
       <!-- Global -->
```

Comment:
If everything is now shown in the same contexts these differentiating comments should be removed.